Quadriga/src/test/java/edu/asu/spring/quadriga/service/publicwebsite/impl/AboutTextManagerTest.java

```diff
@@ -57,20 +85,19 @@ public void testSaveAboutUpdateDTO() throws QuadrigaStorageException {
         AboutTextDTO dto = new AboutTextDTO();
         dto.setId("UniqueId2");
         dto.setProjectId("1st");
-        dto.setTitle("NewTitle");
-        dto.setDescription("NewDescription");
-        Mockito.when(mockedAboutTextDAO.getDTOByProjectId("1st")).thenReturn(dto);
+        dto.setTitle("UpdateTitle");
+        dto.setDescription("Update Description");
+        Mockito.when(mockedAboutTextDAO.generateUniqueID()).thenReturn("UniqueId2");
+        Mockito.when(mockedAboutTextDAO.getDTOByProjectId("PROJxyzwqr")).thenReturn(dto);
 
-        aboutTextManager.saveAbout("1st", "NewTitle2", "NewDescription2");
-
-        Mockito.verify(mockedAboutTextDAO, Mockito.times(0)).generateUniqueID();
+        aboutTextManager.saveAbout("PROJxyzwqr", updateText);
 
         argument = ArgumentCaptor.forClass(AboutTextDTO.class);
         Mockito.verify(mockedAboutTextDAO, Mockito.times(1)).saveOrUpdateDTO(argument.capture());
-        assertEquals("UniqueId2",argument.getValue().getId());
-        assertEquals("1st", argument.getValue().getProjectId());
-        assertEquals("NewTitle2", argument.getValue().getTitle());
-        assertEquals("NewDescription2", argument.getValue().getDescription());
+        assertEquals("UniqueId2", argument.getValue().getId());
+        assertEquals("PROJxyzwqr", argument.getValue().getProjectId());
+        assertEquals("UpdateTitle", argument.getValue().getTitle());
+        assertEquals("Update Description", argument.getValue().getDescription());
```

Comment:
These should be two methods. One that tests save and one that tests update. So that the test results give a clear indication what fails if something files (save might work, but update might fail).